docs/design/performance-benchmarking/SETUP-PERFORMANCE-SUITE.md

```diff
@@ -0,0 +1,11 @@
+# Setup Performance Testing Suite
+
+The performance testing suite is...
```

Comment:
```suggestion
The performance testing suite was developed to support performance tuning of the ODS/API version 3.0
and newer. The ODS Platform team provides primary support for the tools in this test suite. Types of
testing that can run with these tools:

- Volume test: 30 minutes of Create, Update, and Delete requests distributed across many different
  resources. Goal: measure throughput.
- Load test: longer run of the volume tests, with a higher number of clients. Goal: push to breaking point.
- Soak test: 24 hours of volume tests. Goal: measure throughput over a long period of time (is there
  degradation over time?)
- Pipeclean: does every resource work?
- Get All Paging: retrieve all records from all (configured) resources.
```

Reply:
Thanks, I missed that and was supposed to came back to add more detail